corehq/apps/users/models.py

```diff
@@ -1633,26 +1633,7 @@ def analytics_only_case_count(self):
         """
         return CaseData.objects.filter(user_id=self._id).count()
 
-    def location_group_ids(self):
-        """
-        Return generated ID's that represent the virtual
-        groups used to send location data in the restore
-        payload.
-        """
-        def get_sharing_id(loc):
-            return loc.case_sharing_group_object(self._id)._id
-
-        if self.sql_location.location_type.view_descendants:
-            locs = (self.sql_location.get_descendants(include_self=True)
-                                     .filter(location_type__shares_cases=True))
-            return map(get_sharing_id, locs)
-        elif self.sql_location.location_type.shares_cases:
-            return [get_sharing_id(self.sql_location)]
-        else:
-            return []
```

Comment:
only as of this change https://github.com/dimagi/commcare-hq/pull/8485/files#diff-16ce9f2eb3be0695a91eca75714bb3b7R1654